src/fty_common_messagebus_malamute.cc

```diff
@@ -414,6 +423,21 @@ namespace messagebus {
                 }
             }
         }
+        // then try with address of message (= <topic name>)
+        else {
+            const char *address = mlm_client_address(m_client);
+            if (address) {
+                iterator = m_subscriptions.find (address);
+                if (iterator != m_subscriptions.end ()) {
+                    //iterator->second(msg);
+                    std::thread (iterator->second, msg).detach();
```

Comment:
we should not detach thread in the implementation (cf recent merge in the master). Client will do it if he needs / wants

Reply:
Take into account